### PR TITLE
Implement TimeOfDay

### LINKED ISSUE #3: TimeOfDay
Add a `TimeOfDay` type to constrain a `TimeSpan` value to a valid time of day.

### PR DESCRIPTION
**Pull request by slagroom**
- Implements the TimeOfDay type

+semver: feature
close #3

### CODE REVIEW

File: wimm.Secundatives/TimeOfDay.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace wimm.Secundatives
+{
+    /// <summary>
+    /// Represents a time of day.
```
Comment (Insomniak47) [inline wimm.Secundatives/TimeOfDay.cs]: Maybe elaborate a bit here. "Represents a time of day between 0000H and 2359H with no attached tz" etc the summary should say enough.

File: wimm.Secundatives/TimeOfDay.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    /// to <see cref="TimeSpan"/> to be used.
+    /// </remarks>
+    public struct TimeOfDay
+    {
+        private readonly TimeSpan _time;
+
+        public TimeOfDay(TimeSpan time)
+        {
+            if (time < TimeSpan.Zero || time >= TimeSpan.FromHours(24))
+                throw new ArgumentOutOfRangeException(nameof(time), "Must be a positive value less than 24 hours.");
```
Comment (Insomniak47) [inline wimm.Secundatives/TimeOfDay.cs]: Doc this

File: wimm.Secundatives/TimeOfDay.cs (hunk 1 above)
Comment (Insomniak47) [reply wimm.Secundatives/TimeOfDay.cs]: bump

Comment (Insomniak47) [reply wimm.Secundatives/TimeOfDay.cs]: DoubleBump

Comment (Insomniak47) [reply wimm.Secundatives/TimeOfDay.cs]: TrebleBump

**Comment by Insomniak47** [comment]
Buuuuuuump

**Comment by slagroom** [comment]
Jeeez how long does someone have to wait to get a response to a PR update around here?

**Pull request merged**